Tests/UnitTests/Core/MagneticField/ConstantBFieldTests.cpp

```diff
@@ -47,25 +47,7 @@ BOOST_DATA_TEST_CASE(ConstantBField_components,
 
     auto bCache = BField.makeCache(mfContext);
 
-    BOOST_CHECK_EQUAL(Btrue, BField.getField(pos));
-    BOOST_CHECK_EQUAL(Btrue, BField.getField(Vector3(0, 0, 0)));
-    BOOST_CHECK_EQUAL(Btrue, BField.getField(-2 * pos));
-
-    BOOST_CHECK_EQUAL(Btrue, BField.getField(pos, bCache));
```

Comment:
Even if they don't do much, you could technically keep that test since `getField(pos, bCache)` still exist (also true for `BField.getField(Vector3(0, 0, 0), bCache)` and `BField.getField(-2 * pos, bCache)`).

Reply:
My bad I actually missed the fact you removed line 59...
I think now the issue is that if I don't make any mistake we don't test `setField(const Vector3& B)` any more. From what I understand the idea of those test seems to have been : 
- field constructed with a Vector3
- field constructed with 3 double
- field constructed with 3 double and updated with a Vector3
- field constructed with 3 double and updated with a 3 double

Reply:
I think before it only tested construction from 3 doubles, and set with a vector. Honestly I'm tempted to remove the overloads with individual doubles, those are redundant anyway.

Reply:
I agree with you that the overload with 3 double a redundant and could/should probably go. What I wanted to say (sorry if I didn't express myself properly) is that since you have removed line 95 `BField.setField(Btrue);` we don't test for the setter with a vector we only test the setter with 3 double (line 77 of the "new" file) : `BField.setField(bx, by, bz);`

Reply:
Yes. I think it should be tested again now. I also removed the overloads.

Reply:
Yep looks good now